internal/benchrunner/runners/stream/runner.go

```diff
@@ -37,6 +37,8 @@ import (
 	"github.com/elastic/elastic-package/internal/signal"
 )
 
+const numberOfEvents = 10
```

Comment:
beware that 10 events  might not be enough to check an error in a template

for example this template: https://github.com/elastic/integrations/blob/main/packages/aws/_dev/benchmark/rally/ec2metrics-benchmark/template.ndjson#L117-L210

here we render different content according the the value of `$dimensionType`: https://github.com/elastic/integrations/blob/main/packages/aws/_dev/benchmark/rally/ec2metrics-benchmark/config.yml#L4-L12

there is 1/40 possibility that a specific content will be rendered

Reply:
I bumped it to 100, the generation is fairly quick, it shouldn't matter. Is this enough or should we go even higher?